src/ensembl/src/content/app/species-selector/types/loading-state.ts

```diff
@@ -0,0 +1,6 @@
+export enum LoadingState {
+  NOT_REQUESTED = 'not requested',
+  LOADING = 'loading',
+  SUCCESS = 'success',
+  ERROR = 'error'
+}
```

Comment:
This looks like a good loading state enum that can be used across the whole site. So how about moving this enum to a shared space?

Reply:
I'd love to, but am not sure about our shared space conventions. Can we discuss this in more detail?

Reply:
As discussed offline, we should create another pull request for this change.